content/en/docs/gke/deploy/deploy-cli.md

```diff
@@ -49,6 +49,17 @@ one if you haven't already.
 1. Install [Kustomize v3.2.1](https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv3.2.1).
 
     Note, Kubeflow is not compatible with later versions of Kustomize. Read [this GitHub issue](https://github.com/kubeflow/manifests/issues/538) for the latest status.
+    
+    To deploy Kustomize v3.2.1 on a Linux box, follow these commands:
```

Comment:
Suggestion: 

1. Use `run the following commands` instead of `follow these commands`, since we're technically typing them in and then running them. (We, as users, do follow steps though.)

```diff
-    To deploy Kustomize v3.2.1 on a Linux box, follow these commands:
+    To deploy Kustomize v3.2.1 on a Linux box, follow these commands:
...
```

2. Provide a bit more information on how the user should know if the installation went OK. For example:

```diff
-     Then run `kustomize version` to verify the install.
+     Then, to verify the installation, run `kustomize version`. You should see the version of Kustomize in the output if you've successfully deployed it.
```

(while keeping each line at 80 characters max)

WDYT?

Reply:
I agree, the wording is better! Thanks!